2025-07-06 00:44:54,015 INFO    MainThread:14616 [wandb_setup.py:_flush():81] Current SDK version is 0.20.1
2025-07-06 00:44:54,015 INFO    MainThread:14616 [wandb_setup.py:_flush():81] Configure stats pid to 14616
2025-07-06 00:44:54,015 INFO    MainThread:14616 [wandb_setup.py:_flush():81] Loading settings from C:\Users\sghas\.config\wandb\settings
2025-07-06 00:44:54,015 INFO    MainThread:14616 [wandb_setup.py:_flush():81] Loading settings from C:\myProjects\Claims-Severity-Prediction\wandb\settings
2025-07-06 00:44:54,015 INFO    MainThread:14616 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-07-06 00:44:54,015 INFO    MainThread:14616 [wandb_init.py:setup_run_log_directory():703] Logging user logs to C:\myProjects\Claims-Severity-Prediction\wandb\run-20250706_004453-kp1ckixb\logs\debug.log
2025-07-06 00:44:54,025 INFO    MainThread:14616 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to C:\myProjects\Claims-Severity-Prediction\wandb\run-20250706_004453-kp1ckixb\logs\debug-internal.log
2025-07-06 00:44:54,026 INFO    MainThread:14616 [wandb_init.py:monkeypatch_ipython():622] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001E2BF8B1250>
2025-07-06 00:44:54,026 INFO    MainThread:14616 [wandb_init.py:init():831] calling init triggers
2025-07-06 00:44:54,026 INFO    MainThread:14616 [wandb_init.py:init():836] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-07-06 00:44:54,026 INFO    MainThread:14616 [wandb_init.py:init():872] starting backend
2025-07-06 00:44:54,522 INFO    MainThread:14616 [wandb_init.py:init():875] sending inform_init request
2025-07-06 00:44:54,642 INFO    MainThread:14616 [wandb_init.py:init():883] backend started and connected
2025-07-06 00:44:54,652 INFO    MainThread:14616 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2025-07-06 00:44:54,654 INFO    MainThread:14616 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'charmap' codec can't decode byte 0x8d in position 1358: character maps to <undefined>
2025-07-06 00:44:54,655 INFO    MainThread:14616 [wandb_init.py:init():956] updated telemetry
2025-07-06 00:44:54,695 INFO    MainThread:14616 [wandb_init.py:init():980] communicating run to backend with 90.0 second timeout
2025-07-06 00:44:55,744 INFO    MainThread:14616 [wandb_init.py:init():1032] starting run threads in backend
2025-07-06 00:44:58,555 INFO    MainThread:14616 [wandb_run.py:_console_start():2453] atexit reg
2025-07-06 00:44:58,555 INFO    MainThread:14616 [wandb_run.py:_redirect():2301] redirect: wrap_raw
2025-07-06 00:44:59,126 INFO    MainThread:14616 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-07-06 00:44:59,126 INFO    MainThread:14616 [wandb_run.py:_redirect():2393] Redirects installed.
2025-07-06 00:44:59,194 INFO    MainThread:14616 [wandb_init.py:init():1078] run started, returning control to user process
2025-07-06 00:44:59,376 INFO    MainThread:14616 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-07-06 00:50:09,569 INFO    MainThread:14616 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-07-06 00:50:09,570 INFO    MainThread:14616 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-07-06 00:50:09,572 INFO    MainThread:14616 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-07-06 00:50:41,208 INFO    MainThread:14616 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-07-06 00:50:41,208 INFO    MainThread:14616 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-07-06 00:50:41,213 INFO    MainThread:14616 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-07-06 00:50:52,486 INFO    MainThread:14616 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-07-06 00:50:52,494 INFO    MainThread:14616 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-07-06 00:50:52,516 INFO    MainThread:14616 [wandb_run.py:_finish():2219] finishing run gh-saber-technische-universit-t-ilmenau/claim-severity/kp1ckixb
2025-07-06 00:50:52,516 INFO    MainThread:14616 [jupyter.py:save_history():480] not saving jupyter history
2025-07-06 00:50:52,516 INFO    MainThread:14616 [jupyter.py:save_ipynb():404] not saving jupyter notebook
2025-07-06 00:50:52,517 INFO    MainThread:14616 [wandb_init.py:_jupyter_teardown():607] cleaning up jupyter logic
2025-07-06 00:50:52,517 INFO    MainThread:14616 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2025-07-06 00:50:52,517 INFO    MainThread:14616 [wandb_run.py:_restore():2400] restore
2025-07-06 00:50:52,519 INFO    MainThread:14616 [wandb_run.py:_restore():2406] restore done
2025-07-06 00:50:53,433 INFO    MainThread:14616 [wandb_run.py:_footer_history_summary_info():4000] rendering history
2025-07-06 00:50:53,433 INFO    MainThread:14616 [wandb_run.py:_footer_history_summary_info():4032] rendering summary
2025-07-06 00:50:53,433 INFO    MainThread:14616 [wandb_run.py:_footer_sync_info():3961] logging synced files
